crates/sui/src/client_commands.rs

```diff
@@ -610,6 +610,18 @@ pub enum SuiClientCommands {
         address_override: Option<ObjectID>,
     },
 
+    /// Profile the gas usage of a transaction. If an output filepath is not specified, it will output a file `gas_profile_{tx_digest}_{unix_timestamp}.json` which can be opened in a flamegraph tool such as speedscope.
+    #[clap(name = "profile-transaction")]
+    ProfileTransaction {
+        /// The digest of the transaction to replay
+        #[arg(long, short)]
+        tx_digest: String,
+
+        /// If specified, overrides the filepath of the output profile, for example -- /temp/my_profile.json
+        #[clap(name = "profile_output")]
+        profile_output: Option<PathBuf>,
```

Comment:
nit: Can you move the documentation about default output file path to `profile_output`?

```suggestion
    /// Profile the gas usage of a transaction producing a report which can be opened in a flamegraph tool 
    /// such as speedscope.  which can be opened in a flamegraph tool such as speedscope.
    #[clap(name = "profile-transaction")]
    ProfileTransaction {
        /// The digest of the transaction to replay
        #[arg(long, short)]
        tx_digest: String,

        /// File to write the profile output to, defaults to `gas_profile_{tx_digest}_{unix_timestamp}.json` in 
        /// the current working directory.
        #[clap(name = "profile_output")]
        profile_output: Option<PathBuf>,
```

Reply:
This has been resolved, but the comment has not been addressed.